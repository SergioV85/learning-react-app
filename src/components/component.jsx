import React from 'react';

export class Subtitle extends React.Component {
  render() {
    return <h2>This is subtitle, simple component</h2>;
  }
}