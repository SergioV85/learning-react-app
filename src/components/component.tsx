import * as React from 'react';

export class Subtitle extends React.Component {
  public render() {
    return <h2 className="custom-header">This is subtitle, simple component</h2>;
  }
}