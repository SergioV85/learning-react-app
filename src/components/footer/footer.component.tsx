import * as React from 'react';

export const Footer = () =>
  <div className='c-page__footer'>
    <div className='p-2 container'>
      <div className='c-footer'>
        <span className='c-footer__logo'>netflixroulette</span>
      </div>
    </div>
  </div>
  ;
