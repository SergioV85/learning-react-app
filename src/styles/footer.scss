@import './variables';

.c-footer {
  align-items: center;
  display: flex;
  justify-content: space-between;

  &__logo {
    color: $light-red;
    font-family: $tilte-font;
    font-size: 1rem;
    font-weight: 700;
  }
}