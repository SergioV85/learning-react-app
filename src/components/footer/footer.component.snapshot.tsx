import { shallow } from 'enzyme';
import * as React from 'react';
import { Footer } from './footer.component';

describe('Footer Component Snapshot', () => {
  it('should renders correctly', () => {
    const footerSnapshot = shallow(
      <Footer />,
    );
    expect(footerSnapshot)
      .toMatchSnapshot();
  });
});
