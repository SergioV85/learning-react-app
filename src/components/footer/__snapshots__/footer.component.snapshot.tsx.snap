// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Snapshot should renders correctly 1`] = `
<div
  className="c-page__footer"
>
  <div
    className="p-2 container"
  >
    <div
      className="c-footer"
    >
      <span
        className="c-footer__logo"
      >
        netflixroulette
      </span>
    </div>
  </div>
</div>
`;
